I/ImageNoise/Compat.toml

```diff
@@ -0,0 +1,3 @@
+["0.1"]
+Distributions = "0.19"
+julia = ">= 1.0"
```

Comment:
How was this PR made? This is not valid syntax for registry `Compat.toml` files... cc @nkottary, @KristofferC

Reply:
https://github.com/johnnychen94/ImageNoise.jl/commit/a4a34f09bf94c1096215e892d00dfcf8acff0267
through Registrator

Reply:
Ok, in that case this is a Registrator bug—it shouldn't produce invalid Compat syntax.

Reply:
There was a bug in Registrator which was later fixed. Triggering it again should fix it.